src/components/Navigation.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import '../App.css'
+ 
+import * as ROUTES from './routes';
```

Comment:
Nice use of a `ROUTES` object to store the routes. This is a great way to make your life easier down the road. Instead of having to change it tons of places, you can change it in just one!